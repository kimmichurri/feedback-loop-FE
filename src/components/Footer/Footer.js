import React from 'react';

export const Footer = () => {
    return(
      <div className='footer'>
        Turing School of Software and Design, a Colorado Non-Profit Organization.
        Â© Copyright 2019 All Rights Reserved.
      </div>
    )
  
}

export default Footer