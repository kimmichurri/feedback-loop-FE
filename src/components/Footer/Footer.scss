.footer {
  background-color: #2b2b2b;
  text-align: left;
  font-size: .875rem;
  color: #929292;
  padding: 10px;
}
