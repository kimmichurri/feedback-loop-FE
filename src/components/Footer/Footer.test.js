import React from 'react'
import { Footer } from './Footer'
import { shallow } from 'enzyme'

describe('Footer', () => {
  let wrapper
  it('should match the snapshot', () => {
    wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })
})
