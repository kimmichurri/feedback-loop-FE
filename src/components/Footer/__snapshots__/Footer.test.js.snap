// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
<div
  className="footer"
>
  Turing School of Software and Design, a Colorado Non-Profit Organization. Â© Copyright 2019 All Rights Reserved.
</div>
`;
